kotlin/README.md

```diff
@@ -0,0 +1,7 @@
+# Apache Fury™ Kotlin
+
+Provides Fury support for Kotlin Serialization on JVM:
+
+- stdlib `collection`: `ArrayDeque`, `ArrayList`, `HashMap`,`HashSet`, `LinkedHashSet`, `LinkedHashMap`.
```

Comment:
Seems kotlin `ArrayList` is java `ArrayList`, can we clear classes like this?

Reply:
OK! I've split the tested with and implementation section that hopefully clears up the issue!

I plan to add tests for the kotlin primitives. I think out of the box most of them are the same as Java, except for the unsigned types. Then in no particular order I plan to test Array, String, Enum, data class(kotlin equivalent to record) and out of the box stdlib types such as `Pair`, `Triple`.

Reply:
unsigned types are a java object type instead of primitive type in java. We may need to add supports for those types in future, otherwise the serialization for unsigned types will be slow